### PR TITLE
Task/move api prefix to env file

### PR DESCRIPTION
**Pull request by liza-veis**

### CODE REVIEW

File: client/.env.example [hunk 1]
```diff
@@ -3,3 +3,4 @@
 #
 # default. own preference might be used
 REACT_APP_SOCKET_SERVER=http://localhost:3002
+REACT_APP_API_PATH=/api
```
Comment (PavelSukhinin) [inline client/.env.example:6]: `api prefix` maybe?

File: client/src/services/services.js [hunk 2]
```diff
@@ -10,6 +11,7 @@ const storage = new Storage({
 });
 
 const http = new Http({
+  apiPath: ENV.API_PATH,
```
Comment (PavelSukhinin) [inline client/src/services/services.js:14]: FYI, same about prefix

**Review by ghost — APPROVED** [review]

File: client/.env.example (hunk 1 above)
Comment (Liza-Veis) [reply client/.env.example:6]: I named it as in the server. I just called the task before I looked at it

File: client/src/services/services.js (hunk 2 above)
Comment (what1s1ove) [reply client/src/services/services.js:14]: the "path" is okay, because we can pass any link (not just the prefix)

Comment (what1s1ove) [inline client/src/services/services.js:14]: http shouldn't know anything about server. We need to pass it to each service

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: client/.env.example (hunk 1 above)
Comment (PavelSukhinin) [reply client/.env.example:6]: Got it

File: client/src/services/http/http.service.js [hunk 3]
```diff
@@ -62,7 +62,9 @@ class Http {
   }
 
   _getUrl(url, query) {
-    return `${url}${query ? `?${getStringifiedQuery(query)}` : ''}`;
+    const queryString = query ? `?${getStringifiedQuery(query)}` : '';
+
```
Comment (what1s1ove) [inline client/src/services/http/http.service.js:66]: return back

Comment (Liza-Veis) [reply client/src/services/http/http.service.js]: forgot about it

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**